src/language/runtime/values.ts

```diff
@@ -121,7 +122,7 @@ export interface UserClassVal {
     params: ParamDeclaration[]; // for constructor
 }
 
-export type ClassVal = BuiltinClassVal | UserClassVal;
+export type ClassVal = BuiltinClassVal<any> | UserClassVal;
```

Comment:
```suggestion
export type ClassVal = BuiltinClassVal<unknown> | UserClassVal;
```

`any` ist nicht verwendbar, außer du konvertierst existierenden javascript code zu typescript.

Reply:
Ah, danke für den Hinweis!